hazelcast/src/main/java/com/hazelcast/cache/impl/operation/CachePutAllBackupOperation.java

```diff
@@ -88,10 +87,9 @@ public void run() throws Exception {
     private void publishWanEvent(Data key, CacheRecord record) {
         if (cache.isWanReplicationEnabled()) {
             ICacheService service = getService();
-            final CacheWanEventPublisher publisher = service.getCacheWanEventPublisher();
-            final CacheEntryView<Data, Data> view = CacheEntryViews.createDefaultEntryView(
-                    key, toData(record.getValue()), record);
-            publisher.publishWanReplicationUpdate(name, view);
+            CacheWanEventPublisher publisher = service.getCacheWanEventPublisher();
+            CacheEntryView<Data, Data> view = CacheEntryViews.createDefaultEntryView(key, toData(record.getValue()), record);
+            publisher.publishWanReplicationUpdateBackup(name, view);
```

Comment:
We need to send a backup event here, otherwise the new WAN test in EE fails, due to an unexpected additional event, when using the new merge policies.